arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1712 @@
+				"parameters": [{
+					"name": "operationId",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "subscriptionId",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "resourceGroupName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "resourceName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"$ref": "#/parameters/api-version"
+				}, {
+					"$ref": "#/parameters/asyncinfo"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/AsyncOperationResult"
+						}
+					},
+					"default": {
+						"description": "DefaultErrorResponse",
+						"schema": {
+							"$ref": "#/definitions/ErrorDetails"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}/IotHubStats": {
+			"get": {
+				"tags": ["GET"],
+				"summary": "Get IotHub Stats",
+				"description": "Get IotHub Stats",
+				"operationId": "IotHubResource_GetIotHubStats",
+				"consumes": [],
+				"produces": ["application/json", "text/json"],
+				"parameters": [{
+					"name": "resourceName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "subscriptionId",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "resourceGroupName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"$ref": "#/parameters/api-version"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/RegistryStatistics"
+						}
+					},
+					"default": {
+						"description": "DefaultErrorResponse",
+						"schema": {
+							"$ref": "#/definitions/ErrorDetails"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}/skus": {
+			"get": {
+				"tags": ["GET"],
+				"summary": "Get Valid Skus",
+				"description": "Get Valid Skus",
+				"operationId": "IotHubResource_GetValidIotHubSkus",
+				"consumes": [],
+				"produces": ["application/json", "text/json"],
+				"parameters": [{
+					"name": "resourceName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "subscriptionId",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "resourceGroupName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"$ref": "#/parameters/api-version"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/ResponseWithContinuation[IEnumerable[IotHubSkuDescription]]"
```

Comment:
@YuzorMa @yaqiyang - This spec is generated from swashbuckler. Look at  how the plugin is generating the model class for paging ResponseWithContinuation[IEnumerable[IotHubSkuDescription]]. This should actually be IotHubSkuDescriptionListResult with the paging extension applied. 

We should have an onboarding doc for service teams, as to how they can plugin swashbuckle that generates the correct swagger spec as we desire.